edge_computing/image_based_upgrade/images

```diff
@@ -0,0 +1 @@
+../../images/
```

Comment:
In my view, good to mention "Abort" instead of "Cancel" in all images to align with IBU design document.
What do you think?

Reply:
**Original:** "Cancel an ongoing upgrade until the pre-pivot phase in the Upgrade stage"
**Suggestion to change:** "Abort an ongoing upgrade until the pre-pivot phase in the Upgrade stage"

Reply:
Okay. Ack